core/src/main/java/brooklyn/entity/rebind/persister/AbstractBrooklynMementoPersister.java

```diff
@@ -110,6 +115,11 @@ public void delta(Delta delta, PersistenceExceptionHandler exceptionHanlder) {
         memento.updateEnricherMementos(delta.enrichers());
         memento.updateCatalogItemMementos(delta.catalogItems());
     }
+
+    @Override
+    public void queueDelta(Delta delta) {
+        log.warn("Legacy persister ignoring queued delta: "+delta);
```

Comment:
this will be deleted as soon as `0.7.0` is cut.  i'm sure we could delete it now, i'm just being super cautious.  no need to make perfect log messages here!